### ISSUE TITLE
Enhancement

### ISSUE THREAD

**Issue by sarasij190** [kmhmubin/Face-Recognition-Attendance-System#30]
I want to add new features on it in which

1.the admin can calculate the total working hours or active hours of the employee which will show the total active hours and,
2. improve the UI of the system
Can I work on this??

**Comment by ShivamGupta15** [comment]
Yes you are open to work on this as this is open source. Make sure you create a separate branch for your feature like <feature>/<feature-name> branch.
